groups/ecs/locals.tf

```diff
@@ -0,0 +1,23 @@
+locals {
+  secrets     = data.vault_generic_secret.secrets.data
+
+  placement_subnet_cidrs = values(zipmap(
+    values(data.aws_subnet.placement).*.availability_zone,
+    values(data.aws_subnet.placement).*.cidr_block
+  ))
+  placement_subnet_ids = values(zipmap(
+    values(data.aws_subnet.placement).*.availability_zone,
+    values(data.aws_subnet.placement).*.id
+  ))
+  placement_subnet_pattern  = local.secrets.placement_subnet_pattern
+  placement_vpc_pattern     = local.secrets.placement_vpc_pattern
+  automation_subnet_pattern = local.secrets.automation_subnet_pattern
```

Comment:
Though we may need to cater for this later on none of the automation definitions are used. As such we're better off removing them until needed